### PR TITLE
comments.

### PR DESCRIPTION
**Pull request by tompntn**

### CODE REVIEW

File: directions/computeflowfields.go [hunk 1]
```diff
@@ -9,19 +9,26 @@ const (
...
+	Walkable      bool      // If this cell is a wall or not
+	RegionIds     []int     // The region ID's this tile is in
+	Dist          int       // This is only used internally for dijkstra
 }
 
 type MacroMap struct {
-	tiles [][]Tile
+	Width  int
+	Height int
+	tiles  [][]Tile // The tiles.
```
Comment (jdjack) [inline directions/computeflowfields.go]: -_-

File: directions/computeflowfields.go [hunk 2]
```diff
@@ -9,19 +9,26 @@ const (
...
+}
+
+func (mm *MacroMap) GetTile(x, y int) (Tile, error) {
```
Comment (jdjack) [inline directions/computeflowfields.go]: Why does this return an error?

Comment (tompntn) [reply directions/computeflowfields.go]: In case it's not a valid co-ordinate (out of bounds)

File: directions/computeflowfields.go (hunk 1 above)
Comment (tompntn) [reply directions/computeflowfields.go]: :)

**Pull request merged**